runner_manager/models/runner_group.py

```diff
@@ -256,6 +256,25 @@ def healthcheck(
             if runner:
                 log.info(f"Runner {runner.name} created")
 
+    def reset(self, github: GitHub) -> "RunnerGroup":
+        """Reset runner group."""
+        runners = self.get_runners()
+        for runner in runners:
+            if runner.id is not None:
+                github_runner: GitHubRunner = (
+                    github.rest.actions.get_self_hosted_runner_for_org(
+                        self.organization, runner.id
+                    ).parsed_data
+                )
+                runner = runner.update_status(github_runner)
+            if runner.status == RunnerStatus.offline:
+                self.delete_runner(runner)
+                token = github.rest.actions.create_registration_token_for_org(
+                    org=self.organization
+                )
+                self.create_runner(token.parsed_data)
+        return self
```

Comment:
Same comment as above. We already have "self" somewhere else because we call the method on this object itself. What is the point of returning it.